azure-pipeline.yaml

```diff
@@ -108,6 +131,120 @@ stages:
                   displayName: 'GitHub Release (Draft)'
                   inputs:
                     gitHubConnection: 'NEXUSe2e_Github'
-                    assets: '$(Build.ArtifactStagingDirectory)/webapp/*'
+                    assets: '$(Build.ArtifactStagingDirectory)/webapp/*.war'
                     isDraft: true
                     tagSource : 'gitTag'
+  - stage: feed_core
+    displayName: Feed Core Deployment
+    dependsOn: build
+    condition: and(succeeded(), contains(variables['Build.SourceBranch'], 'tags/'))
+    variables:
+      - name: versionTagMatches
+        value: $[ stageDependencies.build.Build.outputs['checkVersion.versionTagMatches'] ]
+    jobs:
+      - deployment: 'Deployment'
+        pool:
+          vmImage: ubuntu-latest
+        environment: 'NEXUSe2e-Devops-Feed'
+        strategy:
+          runOnce:
+            deploy:
+              steps:
+                - powershell: |
+                    echo "Version matches tag: $(versionTagMatches)"
+                    if ("$(versionTagMatches)".Equals("False")) {
+                      Write-Error "Pom versions do not match tag!"
+                    }
+                  displayName: Check Version
+                  name: checkVersion
+                - checkout: self
+                - task: Maven@3
```

Comment:
Muss das Parent nicht auch in den Feed Deployed werden, damit das alles zusammen funktioniert?

Reply:
Parent Version ändert sich nie, die manuelle Version im Feed reicht